index.js

```diff
@@ -24,7 +24,7 @@ class ProfileData {
             `)
     }
 
-    likeRender(){
+    renderLike() {
```

Comment:
names of method mostly show you **Action** and have to start with the verb like DoSmth and etc.